packages/ListBox/src/components/Filter/Filter.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useListBox from "../../useListBox";
+import * as effects from "./effects";
+import { filter, applyFilter } from "./helpers";
+import { FilterContainerStyled, FilterInputStyled, FilterSearchIconStyled } from "./Filter.styles";
+
+const propTypes = {
+  filter: PropTypes.func,
+  filterExcludeSelectedOptions: PropTypes.bool,
```

Comment:
I forget which prefixes we decided to allow... `should`? `will`?  Anyway, as a boolean flag prop I'd suggest something like `shouldExcludeSelections`.

Reply:
this props has been removed is not even part of the component anymore.